test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,56 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind an imported Node.js application to PostgreSQL database in the following order: CSV, DB, App and SBR
+        Given SBO CSV is installed
```

Comment:
SBO CSV is not used during the lifetime of SBO at all. What we do use are CSV of operators managing backing services. In this case it would DB operator.

Reply:
There was no sbo CSV assertion method in e2e so I kept it same other than successful installation in the start.

Reply:
Removed SBO csv.